shanoir-ng-datasets/src/main/java/org/shanoir/ng/datasetacquisition/service/DatasetAcquisitionServiceImpl.java

```diff
@@ -180,4 +180,9 @@ public void deleteById(Long id) throws EntityNotFoundException, ShanoirException
 		shanoirEventService.publishEvent(new ShanoirEvent(ShanoirEventType.DELETE_DATASET_ACQUISITION_EVENT, id.toString(), KeycloakUtil.getTokenUserId(null), "", ShanoirEvent.SUCCESS));
 	}
 
+    @Override
+    public boolean existsByStudyCardId(Long studyCardId) {
+		return repository.existsByStudyCard_Id(studyCardId);
```

Comment:
Not blocking but I looks (at least in GitHub) that the indentation is weird, here and in other files. (I use "hide whitespaces" if it helps)